apps/api/src/domain/models/context/commands/connect-resources-with-note/resources-connected-with-note.event-handler.ts

```diff
@@ -0,0 +1,92 @@
+import {
+    EdgeConnectionMemberRole,
+    IEdgeConnectionContext,
+    IMultilingualText,
+    ResourceCompositeIdentifier,
+} from '@coscrad/api-interfaces';
+import { Inject } from '@nestjs/common';
+import { CoscradEventConsumer, ICoscradEventHandler } from '../../../../../domain/common';
+import { buildMultilingualTextWithSingleItem } from '../../../../../domain/common/build-multilingual-text-with-single-item';
+import { AggregateId } from '../../../../../domain/types/AggregateId';
+import { QUERY_REPOSITORY_PROVIDER_TOKEN } from '../../../shared/common-commands/publish-resource/resource-published.event-handler';
+import { ResourcesConnectedWithNote } from './resources-connected-with-note.event';
+
+// TODO share with API Interfaces?
+export interface IResourceConnectionDto {
+    noteId: AggregateId;
+    compositeIdentifier: ResourceCompositeIdentifier;
+    selfContext: IEdgeConnectionContext;
+    otherContext: IEdgeConnectionContext;
+    // this is the note
+    text: IMultilingualText;
+    // the front-end doesn't currently use this
+    role: EdgeConnectionMemberRole;
+}
+
+export interface IQueryRepositoryForConnectable {
+    /**
+     *
+     * X toMemberCompositeIdentifier
+     * toMemberContext
+     *
+     * X fromMemberCompositeIdentifier
+     * fromMemberContext
+     *
+     * // note- MultilingualText
+     * text: string
+     * languageCode: LangaugeCode
+     */
+    connectResourcesWith(id: string, dto: IResourceConnectionDto): Promise<void>;
```

Comment:
We need to also create an actual edge connection in the graph. I'm going to add this as a second story.